corehq/apps/app_manager/tests/data/fixtures/expected_v2_report.xml

```diff
@@ -0,0 +1,17 @@
+<restore>
+  <filters id="commcare-reports:deadbeef" />
+  <fixture id="commcare-reports:deadbeef" user_id="mock-user-id" report_id="deadbeef" last_sync="2017-09-11T06:35:20" indexed="true">
+    <rows>
+      <row index="0" is_total_row="False">
+        <bar>2</bar>
```

Comment:
Just to be clear: Is this expected to pass or fail?

On the phone this will fail out, since the structure isn't homogenous at the first level, I believe.

Reply:
Ignore the first <restore> tag. That is there because it is needed for this to be a valid XML tag. It'll actually be:

```
<OpenRosaResponse>
  <filters>...</filters>
  <fixture>
    <rows>
      <row>...</row>
    </rows>
  </fixture>
</OpenRosaResponse>

Which should be OK right?

Reply:
that'll be just fine

Reply:
Er, sorry, I think the point of confusion was actually what's _within_ row. That will be a homogenous set, right? Do you have a link to the actual output I can read?

Reply:
@ctsims check out https://staging.commcarehq.org/hq/admin/phone/restore/?as=emord@jemordtest.commcarehq.org&version=2.0 look for fixtures with `commcare-reports`

Reply:
Hey, what is

```
  <filters id="commcare-reports:a1cdf0edec45b8e842eca56e7eba465a"/>
```

in that document? Those aren't something the phone knows how to process...

This fixture from the report
```
  <fixture id="commcare-reports:c73dfed246044bcb8fceda0ba81ff87c" indexed="true" last_sync="2017-10-05T20:09:05.228426" report_id="c73dfed246044bcb8fceda0ba81ff87c" user_id="6ad3ce182215183a4a295b6ef1dffac8">
    <rows>
      <row index="0" is_total_row="False">
        <column_0>emord</column_0>
        <column_1>second test</column_1>
      </row>
      <row index="1" is_total_row="False">
        <column_0>emord</column_0>
        <column_1>test</column_1>
      </row>
    </rows>
  </fixture>
```

seems totally right though.